types/react-portal/index.d.ts

```diff
@@ -21,7 +21,7 @@ export interface PortalFunctionParams {
     isOpen: boolean;
 }
 
-export interface PortalWithStateProps extends React.Props<any> {
```

Comment:
`React.Props` is deprecated. `children` are already typed and wouldn't intersect with a "correct" `ReactNode` that doesn't include `{}`